static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -788,366 +764,83 @@ function WidgetBuilder({
             <MainWrapper>
               <Main>
                 <BuildSteps symbol="colored-numeric">
-                  <BuildStep
-                    title={t('Choose your visualization')}
-                    description={t(
-                      'This is a preview of how your widget will appear in the dashboard.'
-                    )}
-                  >
-                    <DisplayTypeSelector
+                  <VisualizationStep
+                    widget={currentWidget}
+                    organization={organization}
```

Comment:
not sure if we should use the same hook in all subcomponents if this is available in the parent

Reply:
That's a good point. When I thought about this, it was more because I noticed sometimes we pass down props that a nested child component needs to read from, but there are a few layers where nothing happens. I figured `useOrganization` could help us prevent this kind of prop drilling/pass-through by tapping into the context purely for reading, but that could very well make things messy too.

I don't know of any particular opinions on this topic right now so I'd like to do more snooping and learn more, but that's for another day :)